man/figures/link-black-18dp.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="black" width="18px" height="18px"><path d="M0 0h24v24H0z" fill="none"/><path d="M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z"/></svg>
```

Comment:
What about move this file to `inst/rmd/h/anchor-sections`.
We can include this file as an HTML dependency.
In this way, we can render the file correctly even if offline.

Reply:
I looked for that but wasn't sure how to add an svg file (or more generally a resource file) for a CSS usage, as html dependency. Do you have example of that somewhere ?

Reply:
Yes, the example is my initial implementation of anchor sections:
https://github.com/atusy/rmarkdown/blob/323135ef2baea445976427865d92c9d7d19aec35/inst/rmd/h/anchor-sections/anchor-sections.css#L17

Reply:
I missed that - sorry. I was not aware this was correctly copied with the link being modified. I should have trusted **rmarkdown** and **htmltools** to be clever enough. 

It could indeed be interesting to include it. 

Re. offline usage, `html_document()` resulting HTML file requires internet access anyway as I think we load mathjax from a url for now. When you have math expression obviously. Otherwise, everything is fine.

Should we use the link icon that I missed from your PR ? Where does it come from ? The one I used is Apache 2 licensed.

Reply:
Don't worry.

- MathJax requires internet access but falls back to TeX math if offline, right? On the other hand, the svg just disappears if offline.
- The icon comes from pkgdown (<https://github.com/r-lib/pkgdown/blob/master/inst/assets/link.svg>)
- I just noticed that we can add alt text to the CSS `content` (https://developer.mozilla.org/en-US/docs/Web/CSS/content). Will it be helpful if in terms of accessibility? (cc. @jooyoungseo )

Reply:
I'll try to include it. That will also prevent any issue with the link no more working. Thanks for all the information !

Reply:
> * I just noticed that we can add alt text to the CSS `content` (https://developer.mozilla.org/en-US/docs/Web/CSS/content). Will it be helpful if in terms of accessibility? (cc. @jooyoungseo )

Absolutely! That would be very instrumental. `content` alt text is getting supported by and for modern browsers and assistive technologies. Thanks for highlighting that aspect, @atusy!

As a side note concerning Math content, TeX Math is not rendered to be accessible for assitive technologies whereas MathJax is fully accessible. I know it'd be an offline fallback, but having a local copy of MathJax would be ideal when offline for accessibility purpose. (No pressure)

Reply:
Thanks ! 
What would be a useful alt text for that ? Should we use empty alt text so that it is "skipped" ? or should we had a property to ignore the anchor link in screen reader ?

Reply:
@jooyoungseo I am not sure what we should do for this anchor link on sections. 

* What would you expect as alt text for the generated content ? Just a description (like "Anchor svg icon")
* This `content:` alt text is not supported on all browser it seems. 
See https://developer.mozilla.org/en-US/docs/Web/CSS/content#Browser_compatibility
    * Safari and Firefox seem to not support it. I don't know yet if they ignore or if this create an error in the content insertion.

Reply:
@jooyoungseo friendly ping if you can help me on this one ? Thank you

Reply:
I used `aria-label`